.github/workflows/ci.yml

```
@@ -0,0 +1,340 @@
+      with:
+        toolchain: ${{ matrix.rust }}
+        components: clippy, rustfmt
+    
+    - name: Install system dependencies (Ubuntu)
+      if: matrix.os == 'ubuntu-latest'
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libasound2-dev libudev-dev libxcb-render0-dev libxcb-shape0-dev libxcb-xfixes0-dev
+    
+    - name: Cache cargo registry
+      uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)